.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Fixes #[issue number]
+[Please include a summary of the changes and the related issue]
+
+## Demo
+
+[If possible, please include a screenshot or gif/video, it'll make it easier for reviewers to understand the scope of the changes and how the change is supposed to work. If you're introducing something new or changing the existing patterns, please share a Loom and explain what decisions you've made and under what circumstances]
+
+## Release notes
```

Comment:
Should we parse that section in future to automatically generate PR desctiption for changelog?

Reply:
yes, I'd like us to do it at some point